### DISCUSSION TITLE
Tailwind 2 CDN missing colors

### DISCUSSION THREAD

**Discussion by ramjamx** [tailwindlabs/tailwindcss#3458] [Help]
Been utilizing the cdn provided (unpkg) stylesheets for inflight projects that don't need customization/optimization on styles yet, and as of today the stylesheets have lost their colors.

**Comment by simonswiss** [comment]
Hey! Was this a project that you upgraded from Tailwind v1 to Tailwind v2?

Two colours (teal, orange) have been removed from the default colors in Tailwind v2, so if you were relying on those in your project, that would explain why they are gone.

Colors in v2 are not meant to be drop-in replacements, so if that's what you're doing, you should probably create a config file where you bring back the v1 colors and generate Tailwind CSS from that.

We were contemplating the idea of having different versions of Tailwind on CDN, but as of now, you'll need to create your own version of it if you need to use `teal` and `orange`.

Hope it helps 👍
